pkg/neotest/compile.go

```diff
@@ -35,16 +35,21 @@ func CompileSource(t testing.TB, sender util.Uint160, src io.Reader, opts *compi
 	m, err := compiler.CreateManifest(di, opts)
 	require.NoError(t, err)
 
-	return &Contract{
+	c := Contract{
 		Hash:     state.CreateContractHash(sender, ne.Checksum, m.Name),
 		NEF:      ne,
 		Manifest: m,
 	}
+
+	collectCoverage(t, di, c.Hash)
+
+	return &c
 }
 
 // CompileFile compiles a contract from the file and returns its NEF, manifest and hash.
 func CompileFile(t testing.TB, sender util.Uint160, srcPath string, configPath string) *Contract {
 	if c, ok := contracts[srcPath]; ok {
+		collectCoverage(t, rawCoverage[c.Hash].debugInfo, c.Hash)
```

Comment:
We need a documentation for it, please, consider adding a section to https://github.com/nspcc-dev/neo-go/blob/master/pkg/neotest/doc.go.